integration-cli/docker_cli_attach_unix_test.go

```diff
@@ -82,12 +82,7 @@ func (s *DockerSuite) TestAttachAfterDetach(c *check.C) {
 	cpty.Write([]byte{17})
 
 	select {
-	case err := <-errChan:
-		if err != nil {
-			buff := make([]byte, 200)
-			tty.Read(buff)
```

Comment:
😟

Reply:
DId we change the exit code of detach? What exit code is correct?

Reply:
Why the sad face? I added this tty.Read() a while back, but I'm pretty sure it's wrong.

I was seeing `exit status 125` here

Reply:
It's the worried face as in worried that I added it.